addons/point_of_sale/static/src/app/store/order_change_receipt_template.xml

```diff
@@ -2,70 +2,64 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="point_of_sale.OrderChangeReceipt">
-        <div class="pos-receipt">
-            <div class="pos-receipt-order-data"><t t-esc="changes.name" /></div>
-            <t t-if="changes.floor_name || changes.table_name">
-                <br />
+        <div class="pos-receipt m-0 p-0">
+            <!-- Receipt Header -->
+            <div class="receipt-header text-center">
                 <div class="pos-receipt-title">
-                    <t t-esc="changes.floor_name" /> / <t t-esc="changes.table_name"/>
+                    <t t-if="changes.diningModeUpdate">
+                        <t t-if="changes.takeaway">Dine In -&gt; Take Out</t>
+                        <t t-else="">Take Out -&gt; Dine In</t>
+                    </t>
+                    <t t-else="">
+                        <t t-if="changes.takeaway">Take Out</t>
+                        <t t-else="">Dine In</t>
+                    </t>
+                </div>
+                <div class="fs-2">
+                    <span><t t-esc="changes.config_name"/> : <t t-esc="changes.time"/></span><br/>
+                    <span>By: <t t-esc="changes.employee_name"/></span>
                 </div>
-            </t>
-            <div class="pos-receipt-title">
-                <t t-esc="changes.tracking_number"/>
+                <span class="my-4" style="font-size: 120%;">
+                    <t t-if="changes.table_name">Table <strong><t t-esc="changes.table_name"/></strong></t>
+                    <t t-if="changes.tracking_number" class="fw-light my-3"> # <strong><t t-esc="changes.tracking_number"/></strong></t>
+                </span>
             </div>
-            <br />
-            <br />
-            <t t-if="changes.cancelled.length > 0">
-                <div class="pos-order-receipt-cancel">
-                    <div class="pos-receipt-title">
-                        CANCELLED
-                        <t t-esc='changes.time.hours'/>:<t t-esc='changes.time.minutes'/>
-                    </div>
-                    <br />
-                    <br />
-                    <t t-foreach="changes.cancelled" t-as="change" t-key="change_index">
-                        <div class="multiprint-flex">
-                            <span class="product-quantity" t-esc="change.quantity"/>
-                            <span class="product-name" t-esc="change.name"/>
+            <div class="text-center lh-1">---------------------------------------------------------------------</div>
```

Comment:
putting dashes like this is not the best pattern. Can you find a way to use something like `<hr />`?